### PR TITLE
mkdir: add Hindi translation

### PR DESCRIPTION
**Pull request by TheMimikyu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

File: pages.hi/windows/mkdir.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# mkdir
+
+> एक निर्देशिका बनाता है.
```
Comment (kbdharun) [inline pages.hi/windows/mkdir.md:3]:
```suggestion
> एक निर्देशिका बनाता है।
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kbdharun** [comment]
Note: I am closing and reopening the PR to retrigger the CLA check (which needs to pass for the PR to be merged).

**Pull request merged**